### ISSUE TITLE
Missing admin rights to deploy website on Netlifly

### ISSUE THREAD

**Issue by larnsce** [rbind/support#60]
A while ago, I have moved my R-tistic-website repo to [rbind/R-tistic-Blog](https://github.com/rbind/R-tistic-Blog). 

Today, I have committed some changes to the repo and wanted to try deploy the website with [Netlifly](https://app.netlify.com/). I have followed the instructions, but have failed to deploy the website with the following message:

> "We couldn’t set up continuous deployment for this repository. This typically happens when you don’t have admin rights to the repository."

Below also the screenshot:

![netlifly_deployment](https://user-images.githubusercontent.com/18212549/29278533-545c01f6-8115-11e7-91c2-7791e9c98816.png)

I appreciate your support!

**Comment by yihui** [comment]
@thercast @njtierney Could either of you help @larnsce? Thanks!

**Comment by thercast** [comment]
@larnsce you now have admin rights to your repository, let us know if you are still have problems with netlify.

**Comment by larnsce** [comment]
@thercast: Thanks, that worked, but unfortunately I am getting a different error now, which also shows up when I try to build the site locally on my machine.

This, however, is a more general problem not related to Netlifly, but hasn't shown up previously.

After typing `blogdown::build_site()` to into the RStudio Console, I receive the following error message:

```
Error in grep(knitr::all_patterns$md$inline.code, yaml) : 
  invalid regular expression '(?<!(^|
)``)`r[ #]([^`]+)\s*`', reason 'Invalid regexp'
```

Unfortunately, I am unable to identify the source of the error. `blogdown::serve_site` results in the same error message.

**Comment by yihui** [comment]
@larnsce Please reinstall blogdown.

**Comment by larnsce** [comment]
@yihui @thercast: Thanks, works again on my machine, but Netlifly gives the same error. Here the deploy:

```
5:51:24 PM: Build started
5:51:34 PM: Fetching cached dependencies
5:51:34 PM: No cached dependencies found. Cloning fresh repo
5:51:34 PM: git lfs clone git@github.com:rbind/R-tistic-Blog
5:51:37 PM: git remote rm origin
5:51:37 PM: Preparing Git Reference refs/heads/master
5:51:39 PM: Running build command: hugo
5:51:41 PM: v6.11.2 is already installed.
5:51:42 PM: Now using node v6.11.2 (npm v3.10.10)
5:51:42 PM: Using version v6.11.2 of node
5:51:43 PM: Using /opt/buildhome/.rvm/gems/ruby-2.1.2
5:51:43 PM: Started building sites ...
5:51:43 PM: ERROR: 2017/08/15 15:51:43 template.go:132: template: theme/partials/head_highlightjs.html:1:54: executing "theme/partials/head_highlightjs.html" at <.Content>: can't evaluate field Content in type *hugolib.Node in theme/partials/head_highlightjs.html
5:51:43 PM: ERROR: 2017/08/15 15:51:43 general.go:212: Error while rendering taxonomy tag: template: theme/_default/list.html:5:20: executing "theme/_default/list.html" at <.Content>: can't evaluate field Content in type *hugolib.Node
5:51:45 PM: Build complete: exit code: 255
5:51:45 PM: Cleaning up docker container
5:51:45 PM: Error running command: Build script returned non-zero exit code: 255
5:51:45 PM: An error occurred while building the site, skipping the deploy and cache refresh.
5:51:46 PM: Build script returned non-zero exit code: 255
5:51:46 PM: Finished processing build request in 21.881150285s
```

**Comment by yihui** [comment]
@larnsce Make sure your local version of Hugo is the same as the version you use on Netlify.

**Comment by thercast** [comment]
@larnsce just checking in to see if you resolved the issue or would like further assistance?

**Comment by pat-s** [comment]
@thercast @njtierney I also need admin rights for my repository https://github.com/rbind/pat-s_web :)

**Comment by road2stat** [comment]
@pat-s admin added 😎

**Comment by AdminFSP** [comment]
I also need admin right (it's not by default, i'm the owner of the account)

**Comment by road2stat** [comment]
Hi @AdminFSP - is this the repo (at the top of this issue) you are referring to? https://github.com/rbind/R-tistic-Blog

If not, please create a new issue and specify the repo.

If it is: it looks like you have a minimum amount of activities on GitHub. For security reasons, it would be great if we could see some endorsements (to add you as admin) from the current admins of this repo before we can do anything.

**Comment by denisbykovopen** [comment]
I missed my admin rights when remove repo in bitbucket! 
https://bitbucket.org/DenisBykov/givu-dev/src/master/
cannot deploy! please help!
